python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -97,6 +97,24 @@ def keys(*asset_keys: CoercibleToAssetKey) -> "KeysAssetSelection":
         ]
         return KeysAssetSelection(*_asset_keys)
 
+    @public
+    @staticmethod
+    def key_prefixes(*key_prefixes: CoercibleToAssetKey) -> "KeyPrefixesAssetSelection":
+        """Returns a selection that includes assets that match any of the provided key prefixes.
```

Comment:
The docstring should somehow indicate that this accepts lists of strings, in addition to stirngs.

Reply:
added an example